src/AttributeMiddleware.php

```diff
@@ -18,7 +18,8 @@ trait AttributeMiddleware
      */
     public function getMiddleware(): array
     {
-        return array_merge($this->middleware, $this->getMiddlewaresByAttributes());
+        $middlewares = (new ReflectionClass($this))->getAttributes("middleware");
```

Comment:
i need to do this since new base controller laravel doesnt have middleware attributes